app/js/app-menu.js

```diff
@@ -222,6 +222,32 @@ module.exports = function () {
       }
     });
 
+
+    $("#import-simple-af-file").click(function () {
+      $("#import-file-input").trigger('click');
```

Comment:
@kaansancak change `#import-file-input` to `#simple-af-file-input` (in future we will add support for importing files in different formats, hence `#import-file-input` is not a good name).